infrastructure/metadata/infrastructure/testdriver/click_iframe_crossorigin.sub.html.ini

```diff
@@ -2,3 +2,4 @@
   [TestDriver click on a document in an iframe]
     expected:
       if product == "chrome": [PASS, FAIL]  # https://github.com/web-platform-tests/wpt/issues/26295
+      if product == "firefox": [PASS, FAIL]  # https://github.com/web-platform-tests/wpt/issues/45987
```

Comment:
Could you please add an example link to the failure?

Reply:
https://github.com/web-platform-tests/wpt/issues/45987 links to one failure and quotes the relevant logs.